pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/index/ForwardIndexConfig.java

```diff
@@ -150,7 +207,7 @@ public ForwardIndexConfig(@Nullable Boolean disabled, @Nullable CompressionCodec
     }
   }
 
-  @JsonCreator
+  @Deprecated
```

Comment:
I think you can directly remove this constructor. If you want to keep it, given it is no longer than `@JsonCreator`, remove all `@JsonProperty`